### ISSUE TITLE
Style Switcher 

### ISSUE THREAD

**Issue by nadiaff** [MSPARP/newparp#74]
Allows people to switch between accessibility mode, simple mode, black text only mode, pesterchum/trollian chat client mode, and whatever other modes we end up with.

One distinction between normal mode and the chat clients mode is, aside from possible theme change, chat client mode should use acronyms (EB, CG, AC, etc.) and normal mode should use names (John, Karkat, Nepeta, etc.) as the handle that appears preceding the text. There should be an additional field added to Character Creator so that you can enter a shortened name to display preceding the text (in addition to the acronym field).
